src/app/payment/paymentMethod/MollieAPMCustomForm.tsx

```diff
@@ -0,0 +1,122 @@
+import { PaymentMethod } from '@bigcommerce/checkout-sdk';
+import { FieldProps } from 'formik';
+import React, { useCallback, useEffect, useState, FunctionComponent, SyntheticEvent } from 'react';
+
+import { preventDefault } from '../../common/dom';
+import { DropdownTrigger } from '../../ui/dropdown';
+import { FormField } from '../../ui/form';
+
+export interface  MollieCustomCardFormProps {
+    method: PaymentMethod;
+}
+
+interface Issuer {
+    name: string;
+    image: {
+        size1x: string;
+    };
+    id: string;
+}
+
+interface HiddenInputProps extends FieldProps {
+    selectedIssuer?: Issuer;
+}
+
+interface SelecteIssuerProp {
+    selectedIssuer: Issuer;
+}
+
+interface OptionButtonProps {
+    issuer: Issuer;
+    className?: string;
+    onClick?(event: SyntheticEvent<EventTarget>): void;
+}
+
+const MollieAPMCustomForm: FunctionComponent<MollieCustomCardFormProps> = ({ method }) => {
+    const issuers: Issuer[] = method.initializationData?.paymentMethodsResponse;
+
+    const [ selectedIssuer, setSelectedUser ] = useState<Issuer>({ name: 'Select your bank', id: '0', image: { size1x: '' } });
+    const render = useCallback((props: FieldProps) => <HiddenInput { ...props } selectedIssuer={ selectedIssuer } />, [ selectedIssuer ]);
+
+    if (!issuers || issuers.length === 0) {
+        return <></>;
+    }
+
+    const handleClick = ({ currentTarget }: SyntheticEvent<HTMLButtonElement>) => {
+        const _selectedIssuer = issuers.find(({ id }) => id === currentTarget?.dataset.id);
```

Comment:
Why the underscore in `_selectedIssuer`? 🤔

Reply:
because of variable shadowing